### DISCUSSION TITLE
New Sage install on Windows 11 can't find channels

### DISCUSSION THREAD

**Discussion by metapath-zz** [OpenSageTV/Community#91] [Q&A] [answered]
I'm a longtime Sage user and have installed the open source version on several Windows machines with success. For the first time I'm doing it on Windows 11 and have run into a problem.

When I add a video source and scan for channels, no tunable channels are found. Capture device is a 2 tuner Silicon Dust HomeRun (older model that is working fine with my Windows 10 Sage server). The HD config software works fine, finds channels and plays.

What gives? Anybody run into this? I tried installing on another Windows 11 machine and I get the same issue.

**Comment by nyplayer** [comment, marked as answer]
Did You set BDA on the HDHomeRun setup to SageTV ?
![image](https://github.com/OpenSageTV/Community/assets/16063354/8ac5ab12-c1c5-4a3b-984a-16b3c056c239)

**Comment by metapath-zz** [comment]
Bingo! I had selected SageTV during the initial setup, but I just checked it again (must use HD Setup, not Config) and it was set to Default. Changed to Sage, saved, but had to reboot before it worked. I even built a new Win10 install on a spare hard drive and encountered the same issue (when I installed HD software, I only ran Config, not Setup. Sigh.)

Thank you, nyplayer!

**Reply by nyplayer** [reply]
You are Welcome

**Comment by TGsell1** [comment]
Great to see we are all still supporting each other ! For now my setup is still working perfectly.  Win10 server w latest sage release (many years ago) SD guide 4 OTA tuners (silicon dust quad)2 MiniPCs running WMC feeding Uray encoders- OpenDCT connect to Sage.  Mini PCs run win8.1 ‘update frozen’ , tuned from HDHomerun Prime CableCards The real key here is tuning the WMC pcs using the Exe multituner plugin and a script that folks on the old forum helped me write.   I use the mini client with Firesticks for at least 5 TVsThe only thing missing is closed captions.  As long as I have CableCard support I’m good!!  🤞Sent from my iPhoneOn Oct 22, 2023, at 8:46 AM, Metapath ***@***.***> wrote:﻿
Bingo! I had selected SageTV during the initial setup, but I just checked it again (must use HD Setup, not Config) and it was set to Default. Changed to Sage, saved, but had to reboot before it worked. I even built a new Win10 install on a spare hard drive and encountered the same issue (when I installed HD software, I only ran Config, not Setup. Sigh.)
Thank you, nyplayer!
